### DISCUSSION TITLE
Is it possible to set delay for each slide separetly in Slideshow plugin?

### DISCUSSION THREAD

**Discussion by szymonleyk** [igordanchenko/yet-another-react-lightbox#150] [Q&A] [answered]
In Slideshow plugin we have only two properties, and delay is used for each slides. I want to set different delay for each slide. Is it possible?

```jsx
<Lightbox
  open={open}
  close={() => setOpen(false)}
  slides={[...videos, ...slides]}
  slideshow={{ autoplay, delay }}
  plugins={[Slideshow, Video]}
  video={{
    autoPlay: autoplay,
    controls: false,
  }}
/>
```

**Comment by igordanchenko** [comment, marked as answer]
This is not something that is currently supported. Can you provide a use case?

**Reply by szymonleyk** [reply]
Use case:
Create slideshow with 3 images.
First image with 2000 delay
Second with 1000 delay
And the last one with 400 delay

**Reply by igordanchenko** [reply]
Well, that explains the "what", not the "why". And it sounds like a very confusing user experience to me.
